src/Widget/Exception/InvalidConfigException.php

```diff
@@ -0,0 +1,25 @@
+<?php
+declare(strict_types=1);
+
+namespace Yiisoft\Widget\Exception;
+
+use Throwable;
+
+/**
+ * Class InvalidConfigException.
+ */
+class InvalidConfigException extends \Exception
+{
+    /**
+     * __construct.
+     *
+     * @param string $message
+     * @param integer $code
+     *
+     * @param Throwable $previous
+     */
+    public function __construct(string $message, int $code = 0, Throwable $previous = null)
+    {
+        parent::__construct($message, $code, $previous);
+    }
```

Comment:
Can't you avoid this? \Exception class has the same constructor

https://www.php.net/manual/en/exception.construct.php

Reply:
Done.